src/js/domain/Lotto.js

```diff
@@ -60,8 +56,12 @@ class Lotto {
     return [...lottoNumbers].sort((a, b) => a - b);
   }
 
-  hasLottoNumberInstance(lottoNumberInstance) {
-    return this.#lottoNumberInstances.includes(lottoNumberInstance);
+  contains(lottoNumber) {
+    if (lottoNumber instanceof LottoNumber) {
+      return this.#lottoNumbers.includes(lottoNumber);
+    }
+
+    return this.lottoNumberValues.includes(lottoNumber);
   }
```

Comment:
```suggestion
  contains(lottoNumber) {
    return this.lottoNumberValues.includes(new LottoNumber(lottoNumber));
  }
```
이렇게 해도 무방해보이네요! 어차피 LottoNumber에서 동일한 값을 반환하도록 보장해주니까요 ㅎㅎ